### PR TITLE
Use C type casting instead of static_cast.

### PR DESCRIPTION
**Pull request by rtakacs**

### CODE REVIEW

File: targets/esp8266/include/jerry_extapi.h [hunk 1]
```diff
@@ -28,10 +28,10 @@
 
 #define JS_VALUE_TO_NUMBER(val_p) \
     ((val_p)->type == JERRY_API_DATA_TYPE_FLOAT32 ? \
-       static_cast<double>((val_p)->v_float32) : \
+       (double)((val_p)->v_float32) : \
```
Comment (LaszloLango) [inline targets/esp8266/include/jerry_extapi.h]: Strange indentation. Can you write this in one line or align the colon and the question mark to the same column?

File: targets/esp8266/include/jerry_extapi.h [hunk 2]
```diff
@@ -28,10 +28,10 @@
...
      (val_p)->type == JERRY_API_DATA_TYPE_FLOAT64 ? \
-       static_cast<double>((val_p)->v_float64) : \
-     static_cast<double>((val_p)->v_uint32))
+       (double)((val_p)->v_float64) : \
```
Comment (LaszloLango) [inline targets/esp8266/include/jerry_extapi.h]: ditto

File: targets/mbedk64f/source/jerry_extapi.h [hunk 3]
```diff
@@ -30,10 +30,10 @@
 
 #define JS_VALUE_TO_NUMBER(val_p) \
     ((val_p)->type == JERRY_API_DATA_TYPE_FLOAT32 ? \
-       static_cast<double>((val_p)->v_float32) : \
+       (double)((val_p)->v_float32) : \
```
Comment (LaszloLango) [inline targets/mbedk64f/source/jerry_extapi.h]: ditto. Is this file a duplicate of `esp8266/include/jerry_extapi.h`?

**Comment by ruben-ayrapetyan** [comment]
Looks good to me

**Comment by rtakacs** [comment]
@LaszloLango: I've updated the PR.

**Comment by LaszloLango** [comment]
@rtakacs, thanks. LGTM

**Comment by LaszloLango** [comment]
Landed (efc994b112e30cd0a20f8a2365ea94c84bec0eec)

**Pull request closed without merging**